helm-chart/README.md

```diff
@@ -109,7 +109,8 @@ helm install --name my-release -f myvalues.yaml .
 | `connection.port`                 | Port the db listens to                      | `5432`                             |
 | `connection.dbName`               | Database name in TimescaleDB to connect to  | `timescale`                        |
 | `connection.sslMode`              | SSL mode for connection                     | `require`                          |
-| `service.port`                    | Port the connector pods will accept connections on | `9201`                      |
```

Comment:
We should probably mention that we are breaking backwards compatibility with this

Reply:
Added it to the draft release notes. As we discussed.